### PR TITLE
Documentation/cisco identity configuration

### PR DESCRIPTION
**Pull request by fdurand**
# Description

Describe the new cisco Identity Networking Policy
# Impacts

No
# Delete branch after merge

YES
# NEWS file entries
## Enhancements
- Documentation to configure cisco switch with Identity Networking Policy

### CODE REVIEW

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 1]
```diff
@@ -1425,6 +1425,335 @@ PacketFence switch configuration:
...
+   mab
+   access-session host-mode single-host
+   access-session control-direction in
+   access-session closed
+   access-session port-control auto
+   authentication periodic
+   authentication timer reauthenticate server
+   service-policy type control subscriber MACAUTH
+
+Interface Template (801.1x):
```
Comment (julsemaan) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: Wow 801.1x thats outdated, I heard 803.1x is coming out next week.

We need to keep up

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 2]
```diff
@@ -1425,6 +1425,335 @@ PacketFence switch configuration:
 
 NOTE: AAA authentication is slow to come up after a reload of the IOS XE switches. This makes the recovery from a reboot longer to complete. This is due to a bug in IOS XE. A workaround is to execute the following command `no aaa accounting system default start-stop group tacacs+`.
 
+Identity Networking Policy
+++++++++++++++++++++++++++
+
+Starting from version 15.2(1)E (IOS) and 3.4.0E (IOSXE) , Cisco introduce the Identity Based Networking Services.
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: introduced

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 3]
```diff
@@ -1425,6 +1425,335 @@ PacketFence switch configuration:
...
+It mean that you can create a authentication workflow on the switch and create interfaces templates.
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: It means .. create an ...

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 4]
```diff
@@ -1425,6 +1425,335 @@ PacketFence switch configuration:
...
+  aaa new-model 
+  aaa group server radius packetfence 
+   server name packetfence
+  !
+  aaa authentication login default local 
+  aaa authentication dot1x default group packetfence 
+  aaa authorization network default group packetfence 
+  radius-server vsa send authentication
+
+Radius server configuration:
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: RADIUS server

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 5]
```diff
@@ -1425,6 +1425,335 @@ PacketFence switch configuration:
...
+   key useStrongerSecret
+   automate-tester username cisco idle-time 1
+
+CoA configuration:
+
+  aaa server radius dynamic-author
+   client 192.168.1.5 server-key useStrongerSecret
+  port 3799
+
+Activate SNMP on the switch:
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: Activate -> Enable    -- same thing everywhere

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 6]
```diff
@@ -1425,6 +1425,335 @@ PacketFence switch configuration:
...
+
+  snmp-server community public RO
+
+Activate http and https server
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: HTTP and HTTPS end with semi-colon

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 7]
```diff
@@ -1425,6 +1425,335 @@ PacketFence switch configuration:
...
+
+  ip http server
+  ip http secure-server
+
+Activate ip device tracking
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: ip -> IP and end with semi-colon

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 8]
```diff
@@ -1425,6 +1425,335 @@ PacketFence switch configuration:
...
+ !
+ class-map type control subscriber match-all MAB_FAILED
+ match method mab
+ match result-type method mab authoritative
+ !
+ class-map type control subscriber match-all DOT1X_FAILED
+ match method dot1x
+ match result-type method dot1x authoritative
+
+Policy Map:
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: Map -> map

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 9]
```diff
@@ -1425,6 +1425,335 @@ PacketFence switch configuration:
...
+
+On the 3 following configurations if the radius server is down then we will apply
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: RADIUS

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 10]
```diff
@@ -1425,6 +1425,335 @@ PacketFence switch configuration:
...
+ CRITICAL_AUTH_VLAN, DEFAULT_CRITICAL_VOICE_TEMPLATE and CRITICAL-ACCESS service template.
+If the radius server goes up then it reinitialize the authentication if the port is in
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: RADIUS ... reinitializes

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 11]
```diff
@@ -1425,6 +1425,335 @@ PacketFence switch configuration:
...
+IN_CRITICAL_VLAN.
+
+for 802.1X with Mac Authentication fallback:
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: MAC

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 12]
```diff
@@ -1425,6 +1425,335 @@ PacketFence switch configuration:
...
+   20 class NOT_IN_CRITICAL_AUTH do-until-failure
+    10 resume reauthentication
+  event inactivity-timeout match-all
+   10 class always do-until-failure
+    10 clear-session
+  event authentication-success match-all
+   10 class always do-until-failure
+    10 activate service-template DEFAULT_LINKSEC_POLICY_SHOULD_SECURE
+
+for Mac Authentication only:
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: MAC

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 13]
```diff
@@ -1425,6 +1425,335 @@ PacketFence switch configuration:
...
+    20 class NOT_IN_CRITICAL_AUTH do-until-failure
+     10 resume reauthentication
+   event inactivity-timeout match-all
+    10 class always do-until-failure
+     10 clear-session
+   event authentication-success match-all
+    10 class always do-until-failure
+     10 activate service-template DEFAULT_LINKSEC_POLICY_SHOULD_SECURE
+
+for 802.1X Only:
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: only

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 14]
```diff
@@ -1425,6 +1425,335 @@ PacketFence switch configuration:
...
+    20 class NOT_IN_CRITICAL_AUTH do-until-failure
+     10 resume reauthentication
+   event inactivity-timeout match-all
+    10 class always do-until-failure
+     10 clear-session
+   event authentication-success match-all
+    10 class always do-until-failure
+     10 activate service-template DEFAULT_LINKSEC_POLICY_SHOULD_SECURE
+
+Interface Template (802.1x Mac Authentication):
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: 802.1X MAC Authentication

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 15]
```diff
@@ -1425,6 +1425,335 @@ PacketFence switch configuration:
...
+   mab
+   access-session host-mode multi-domain
+   access-session control-direction in
+   access-session closed
+   access-session port-control auto
+   authentication periodic
+   authentication timer reauthenticate server
+   service-policy type control subscriber DOT1X_MAB
+
+Interface Template (Mac Authentication):
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: MAC

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 16]
```diff
@@ -1425,6 +1425,335 @@ PacketFence switch configuration:
...
+   mab
+   access-session host-mode single-host
+   access-session control-direction in
+   access-session closed
+   access-session port-control auto
+   authentication periodic
+   authentication timer reauthenticate server
+   service-policy type control subscriber MACAUTH
+
+Interface Template (802.1x):
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: 802.1X

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 17]
```diff
@@ -1425,6 +1425,335 @@ PacketFence switch configuration:
...
+   mab
+   access-session host-mode single-host
+   access-session control-direction in
+   access-session closed
+   access-session port-control auto
+   authentication periodic
+   authentication timer reauthenticate server
+   service-policy type control subscriber DOT1X
+
+On each interface for 802.1x with Mac Authentcation:
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: 802.1X ... MAC

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 18]
```diff
@@ -1425,6 +1425,335 @@ PacketFence switch configuration:
...
+
+  source template identity-template-mab
+  dot1x timeout tx-period 5
+
+On each interface for Mac Authentication:
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: MAC

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 19]
```diff
@@ -1425,6 +1425,335 @@ PacketFence switch configuration:
...
+
+  source template identity-template-macauth
+
+On each interface for 802.1x:
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: 802.1X

**Pull request merged**